src/js/core/widget/core/OnOffSwitch.js

```diff
@@ -386,6 +399,136 @@
 
 				if (self._type === "select") {
 					element.selectedIndex = value;
+					self._ui.input.checked = !!value;
+				}
+			};
+
+			OnOffSwitch.prototype._onDragStart = function () {
+				var self = this,
+					ui = self._ui,
+					checkbox = ui.input,
+					handler = ui.handler,
+					inputRect = checkbox.getBoundingClientRect(),
+					handleRect = handler.getBoundingClientRect(),
+					inputStyle = window.getComputedStyle(checkbox),
+					border = parseInt(inputStyle.borderLeftWidth, 10) +
+						parseInt(inputStyle.borderLeftWidth, 10);
```

Comment:
Is it correct to add the borderLeftwidth twice?

Reply:
You are right. Thanks.

Reply:
Done.